hieradata/development.yaml

```diff
@@ -105,9 +103,6 @@ govuk::apps::collections_publisher::enable_procfile_worker: false
 govuk::apps::contacts::extra_aliases: ['contacts-admin']
 govuk::apps::content_store::mongodb_nodes: ['localhost']
 govuk::apps::content_store::mongodb_name: 'content_store_development'
-govuk::apps::content_performance_manager::rabbitmq_user: 'content_performance_manager'
-govuk::apps::content_performance_manager::rabbitmq_hosts: ['localhost']
-govuk::apps::content_performance_manager::rabbitmq::amqp_pass: 'content_performance_manager'
```

Comment:
Same here - shouldn't we have these keys for content-data-api?

Reply:
I've just removed things here. Some additional setup is needed for the Content Data API in the development-vm, but that's covered here https://github.com/alphagov/govuk-puppet/pull/9091/files#diff-ff48ae5906656c3a1014d9acb3b0abdcR108